ENGAGEHF/Dashboard/Notifications/Views/NotificationRow.swift

```diff
@@ -0,0 +1,111 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct NotificationRow: View {
+    @Binding var notification: Notification
+    @State private var isExpanded = false
+    
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack(alignment: .center) {
+                Text(notification.type)
+                    .font(.subheadline)
+                    .foregroundStyle(.secondary)
+                Spacer()
+                XButton(notification: $notification)
+            }
+            Divider()
+            Text(notification.title)
+                .multilineTextAlignment(.leading)
+                .padding(.bottom, 10)
+            if isExpanded {
+                Text(notification.description)
+                    .multilineTextAlignment(.leading)
+                    .font(.footnote)
+            } else {
+                LearnMoreButton(isExpanded: $isExpanded)
+            }
```

Comment:
I am wondering if we should show at least one line and expand it to more lines (if available) using the learn more button. E.g. we can use the line limit modifier on a `Text` to set the line limit from 1 to 0 (I think that indicates that there is no line limit). So we have three states:

1. The Text fits in a single line (we could determine this like in this answer here: https://stackoverflow.com/a/75827200) and therefore there is no need for an expand button.
2. The Text doesn't fit in a single line and we are not in an expanded state: We show the text with a line limit of 1 and we show the show more button.
3. The Text doesn't fit in a single line and we are in an expanded state: We just show the text without any line limit.